README.md

```diff
@@ -21,16 +21,16 @@
 
 [![CodeQL](https://github.com/github/issue-metrics/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/github/issue-metrics/actions/workflows/codeql-analysis.yml) [![Docker Image CI](https://github.com/github/issue-metrics/actions/workflows/docker-image.yml/badge.svg)](https://github.com/github/issue-metrics/actions/workflows/docker-image.yml) [![Python package](https://github.com/github/issue-metrics/actions/workflows/python-package.yml/badge.svg)](https://github.com/github/issue-metrics/actions/workflows/python-package.yml)
 
-This is a GitHub Action that searches for pull requests/issues/discussions in a repository and measures and reports on
-several metrics. The issues/pull requests/discussions to search for can be filtered by using a search query.
+This is a GitHub Action that searches for issues/pull requests/discussions in a repository, measures several metrics, and generates a report in form of a GitHub issue.
+The issues/pull requests/discussions to search for can be filtered by using a search query.
 
 The metrics that are measured are:
 | Metric | Description |
 |--------|-------------|
 | Time to first response | The time between when an issue/pull request/discussion is created and when the first comment or review is made.* |
 | Time to close | The time between when an issue/pull request/discussion is created and when it is closed.* |
 | Time to answer | (Discussions only) The time between when a discussion is created and when it is answered. |
-| Time in label | The time between when a label has a specific label applied to an issue/pull request/discussion and when it is removed. This requires the LABELS_TO_MEASURE env variable to be set. |
+| Time in label | The time between when a label has a specific label applied to an issue/pull request/discussion and when it is removed. This requires the `LABELS_TO_MEASURE` env variable to be set. |
 
 *For pull requests, these metrics exclude the time the PR was in draft mode.
 *For Issue and pull requests, issue/pull request author's own comments and comments by bots are excluded.
```

Comment:
I assume that these are the footnotes, explaining more details about the `*` in row 1 and row 2 of the table above?

I was not sure:
- if these are two separate footnotes, or if both footnotes apply to both rows?
- if the footnotes are supposed to be rendered in a single row (as they are now)? it looks a bit strange this way.

Screenshot:

![Screenshot 2023-10-10 at 23 51 24](https://github.com/github/issue-metrics/assets/163029/5ea7e412-caf5-466c-8d0e-1f1a80843969)

Reply:
Yeah, these are supposed to be rendered in separate rows 😓